tests/kubernetes/test_pod_generator.py

```diff
@@ -658,11 +658,12 @@ def test_deserialize_model_file(self):
     def test_pod_name_confirm_to_max_length(self, _, pod_id):
         name = PodGenerator.make_unique_pod_id(pod_id)
         assert len(name) <= 253
-        parts = name.split(".")
-        if len(pod_id) <= 63:
+        parts = name.split("-")
+        if len(pod_id) <= 63 - 33:
             assert len(parts[0]) == len(pod_id)
         else:
             assert len(parts[0]) <= 63
```

Comment:
```suggestion
        assert parts[0] == pod_id[:30]
```

Did this get lost in the shuffle?

Reply:
chagned to `assert len(parts[0]) <= 30`. The prefix can be less than 30, but not larger.